### DISCUSSION TITLE
About k-35 actual size

### DISCUSSION THREAD

**Discussion by RexLAz** [Chia-Network/chia-blockchain#2716] [Q&A] [answered]
I prepared a 6.4Tb(Actual size 5.82Tb) ssd for k35 farming, it shows on the GUI each k35 temp plot file should be 2175Gib. I was doing 2 of them it, total temp file should be 4350Gib. It should be over 1500Gb left in the SSD .
But when the plot progress in 33%, it shows there is only 469gb space left  in my SSD. I am wondering is the GUI shows the wrong temp size or something ? Can someone explain to me ? should i call it off or just let the plot keep going? 
我准备了一个6.4 T的SSD 实际大小5.82tb 用来P K35的图，两个临时文件按照GUI上的显示一共 才4350Gib 按道理应该还有 1.5Tb左右的富裕空间，但是在进行到33%的时候，我发现SSD里只有不到500GB的剩余空间，我是应该继续让P图进行呢 还是格式化了呢？

**Comment by moshenskyDV** [comment, marked as answer]
Relax, 32% it's a peak of memory usage. after third stage start (33-34%) it will free a little, and memory usage goes down.

**Comment by heartlesstakei** [comment]
Please do refer to this https://github.com/Chia-Network/chia-blockchain/wiki/k-sizes

**Comment by cyr616** [comment]
等就行了
